src/RemoteAPI/class-remote-api-base.php

```diff
@@ -134,4 +155,25 @@ public function get_items( $query, $associative = false ) {
 
 		return $associative ? convert_to_associative_array( $response ) : $response;
 	}
+
+	/**
+	 * Check if user is capable for making API call.
```

Comment:
I think this occurrence (and additional occurrences) can be improved to:

```suggestion
	 * Checks if the current user is allowed to make the API call.
```